### PR TITLE
Add day 4 schedule

### PR DESCRIPTION
**Pull request by selynna**

### CODE REVIEW

File: 2016/playbook/session_2/SCHEDULE.md [hunk 1]
```diff
@@ -81,7 +81,23 @@
...
 ## Day 4
 
-On day 4 we'll be running the [Dodge][dodge] workshop.
+| Time     | Activity                                           |
+| -------- | -------------------------------------------------- |
+| 8:40 AM  | Standups                                           |
+| 9:00 AM  | Hacker arrival                                     |
+| 9:10 AM  | Energizer 1                                        |
+| 9:25 AM  | Breakfast                                          |
+| 9:45 AM  | [Dodge][dodge]          						    |
```
Comment (maxwofford) [inline 2016/playbook/session_2/SCHEDULE.md]: Would you please reformat this table?

Comment (selynna) [inline 2016/playbook/session_2/SCHEDULE.md]: Didn't show up in my text editor, thanks for pointing that out

File: 2016/playbook/session_2/SCHEDULE.md [hunk 2]
```diff
@@ -87,12 +87,12 @@
...
 | 9:25 AM  | Breakfast                                          |
-| 9:45 AM  | [Dodge][dodge]          						    |
+| 9:45 AM  | [Dodge][dodge]          						                |
 | 12:30 PM | Lunch                                              |
 | 1:30 PM  | Energizer 2                                        |
-| 2:00 PM  | [Dodge (continued)][dodge] 						|
+| 2:00 PM  | [Dodge (continued)][dodge] 						            |
 | 3:30 PM  | [Energizer 3](../ACTIVITIES.md#walking)            |
-| 3:45 PM  | [Dodge (continued)][dodge] 						|
+| 3:45 PM  | [Dodge (continued)][dodge] 					            	|
```
Comment (maxwofford) [inline 2016/playbook/session_2/SCHEDULE.md]: did you mean to do this?

File: 2016/playbook/session_2/SCHEDULE.md [hunk 3]
```diff
@@ -81,7 +81,23 @@
...
+| -------- | -------------------------------------------------- |
+| 8:40 AM  | Standups                                           |
+| 9:00 AM  | Hacker arrival                                     |
+| 9:10 AM  | Energizer 1                                        |
+| 9:25 AM  | Breakfast                                          |
+| 9:45 AM  | [Dodge][dodge]          						                |
+| 12:30 PM | Lunch                                              |
+| 1:30 PM  | Energizer 2                                        |
+| 2:00 PM  | [Dodge (continued)][dodge] 						            |
+| 3:30 PM  | [Energizer 3](../ACTIVITIES.md#walking)            |
```
Comment (maxwofford) [inline 2016/playbook/session_2/SCHEDULE.md:94]: thoughts on having the walk for 30 minutes?

Comment (selynna) [inline 2016/playbook/session_2/SCHEDULE.md:94]: Sure, we could do that. Will change now

File: 2016/playbook/session_2/SCHEDULE.md (hunk 2 above)
Comment (selynna) [inline 2016/playbook/session_2/SCHEDULE.md]: Idk what's up with my table formatting, it all looks right when I do it in Sublime and in the GitHub browser editor

**Comment by selynna** [comment]
Done

**Pull request closed without merging**